### PR TITLE
web: Add env variable for setting up the frontend port

### PR DESCRIPTION
**Pull request by bidlako**
Replacing the hardcoded port with env variable.

### CODE REVIEW

File: web/setupProxy.js [hunk 1]
```diff
@@ -22,6 +24,6 @@ router.get('/healthcheck', function (req, res) {
 
 app.use(router)
 
-app.listen(3000, function() {
-  console.log('App listening on port 3000!')
+app.listen(port, function() {
```
Comment (phixMe) [inline web/setupProxy.js:27]: Is there a default we need to set for this?

**Review by wslulciuc — APPROVED** [review]
LGTM 👍

**Comment by wslulciuc** [comment]
@bidlako, changes look good to me, but to merge, you'll need to sign your [commits](https://github.com/MarquezProject/marquez/pull/2838/checks?check_run_id=28579025247).

**Review by phixMe — APPROVED** [review]
I pulled this down and verified, thanks for the change.

**Pull request merged**